### DISCUSSION TITLE
"get" should be "aws" on "Deploying to AWS" page

### DISCUSSION THREAD

**Discussion by tmintz** [l3atbc-datadog/pushkin#105] [General] [answered]
I think there's a typo after the section on installing the ECS CLI. The commands for obtaining the keys are given as:
```
$ get configure get aws_access_key_id --profile [profile]
$ get configure get aws_secret_access_key --profile [profile]
```
but I believe it should be
```
$ aws configure get aws_access_key_id --profile [profile]
$ aws configure get aws_secret_access_key --profile [profile]
```
(Mac users who installed aws system-wide will have to prefix "sudo" to the commands)

**Comment by jkhartshorne** [comment]
@tmintz since you are a power-user / early-adopter, can I ask you to submit bugs via Github Issues? If you look at the menu above, you'll see we are on the "Discussions" tab. Two tabs to the left should be "Issues". That's what we use as our todo list for fixing things. Thanks Toby! And thanks for catching this.

(We'll fix it whether or not you submit an issue, but I believe you get a github badge for submitting an issue. Plus I want to get people used to this method.)

**Reply by tmintz** [reply]
Sure thing!  Wasn't sure whether it was ok to post there.

**Comment by jkhartshorne** [comment]
Absolutely ok! I didn't mention it because I figured (correctly) that most
questions would just be people confused about this or that. And that's
better in the Discussion forum. But for people who know when they've
(probably) found a bug, it's more streamlined if you go straight for
Issues.

**Comment by conbainbridge** [comment, marked as answer]
Great catch, this has been updated!
